src/hotspot/src/cpu/aarch64/vm/c1_LIRAssembler_aarch64.hpp

```diff
@@ -24,8 +24,8 @@
  *
  */
 
-#ifndef CPU_X86_VM_C1_LIRASSEMBLER_X86_HPP
-#define CPU_X86_VM_C1_LIRASSEMBLER_X86_HPP
+#ifndef CPU_X86_VM_C1_LIRASSEMBLER_AARCH64_HPP
+#define CPU_X86_VM_C1_LIRASSEMBLER_AARCH64_HPP
```

Comment:
CPU is still x86, which doesn't make sense.

Reply:
Good catch, will fix.